spec/dummy/config/routes.rb

```diff
@@ -6,5 +6,7 @@
   # Defines the root path route ("/")
   # root "articles#index"
 
+  get 'show-content-in-hovercard', to: 'hovercard#show' if Rails.env.development?
```

Comment:
no lo creo necesario ya que es el dummy app. Nunca la vamos a correr en producción

```suggestion
  get 'show-content-in-hovercard', to: 'hovercard#show'
```